web/atlas.json

```diff
@@ -5616,7 +5596,6 @@
 {"id": "txefbr", "submitted_by": "xVandherLord", "name": "TheBichoZombie", "description": "Mascot representing a small spanish-speaking community on twitch.", "website": "https://www.twitch.tv/thebichozombie", "subreddit": "", "center": [1373.5, 463.5], "path": [[1364.5, 466.5], [1367.5, 469.5], [1379.5, 469.5], [1381.5, 467.5], [1381.5, 461.5], [1378.5, 461.5], [1378.5, 455.5], [1367.5, 455.5], [1367.5, 461.5], [1364.5, 461.5], [1364.5, 467.5]]},
 {"id": "txef4v", "submitted_by": "JoeBobTNVS", "name": "Super Mario Pictograms", "description": "A mini-mushroom and invincibility star from the Mario franchise", "website": "https://mario.nintendo.com/", "subreddit": "/r/nintendo", "center": [878.5, 1859.5], "path": [[872.5, 1847.5], [872.5, 1870.5], [884.5, 1870.5], [884.5, 1847.5], [872.5, 1847.5]]},
 {"id": "txeepb", "submitted_by": "ININHAHALELE", "name": "Moona Hoshinova", "description": "Moona Hoshinova (1st Gen.) is a talent from the Idol VTuber group, Hololive. She is the first Hololive Talent from Indonesia to reach a milestone of a million subscribers.", "website": "https://hololive.hololivepro.com/en/talents?gp=indonesia", "subreddit": "/r/hololive", "center": [1384.5, 995.5], "path": [[1381.5, 990.5], [1386.5, 990.5], [1389.5, 998.5], [1379.5, 998.5], [1380.5, 992.5], [1383.5, 994.5], [1384.5, 993.5], [1388.5, 996.5], [1388.5, 997.5], [1387.5, 996.5], [1382.5, 996.5]]},
-{"id": "txeej9", "submitted_by": "AsukaMainGG", "name": "Mana Nagase", "description": "Mana Nagase was an idol hailing from the Japanese idol franchise IDOLY PRIDE. She was voiced by the late Sayaka Kanda.\n \nShe currently has four songs (First Step, Precious, Memories of the Starry Sky, and Song For You) and two covers (Racing into the Night by YOASOBI and Ghost in a Flower by Yorushika)", "website": "", "subreddit": "/r/IdolyPride", "center": [1497.5, 1242.5], "path": [[1489.5, 1234.5], [1489.5, 1249.5], [1505.5, 1249.5], [1505.5, 1234.5]]},

```

Comment:
This has info that the other description doesn't. Merge the two descriptions to preserve that info.